doc-tool/src/dotty/tools/dottydoc/staticsite/Site.scala

```diff
@@ -311,6 +315,16 @@ case class Site(val root: JFile, val projectTitle: String, val documentation: Ma
     * defaults, the user-defined one will take precedence.
     */
   val layouts: Map[String, Layout] = {
+
+    val cal = Calendar.getInstance()
```

Comment:
I would recommend against using java.util.Date/Calendar/SimpleDataFormat. These APIs have many pitfalls and are not thread-safe.  Java 8 introduced a new set of APIs to deal with date and time in the `java.time` package that should be used instead. To get the current time you should use LocalDateTime.now: https://docs.oracle.com/javase/8/docs/api/java/time/LocalDateTime.html#now--